contrib/morse2beep

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/perl
+# Turn Morse code into beeps.
```

Comment:
(GPLv2 or MIT or Unlicense are fine, other license to be seen.)

Reply:
I usually just plaster in
`# Copyright       : https://www.fsf.org/copyleft/gpl.html`
but perhaps you could stick a better one in there for me. Thanks!

Reply:
Also I suppose my Date and Version headers won't make too much sense now that editing has moved to GitHub. So maybe you can strip them. Thanks.

Reply:
So, changing the comment prefix from the `*.[ch]` " * " to "# ", would the following work for you?

```
# Copyright 2020 Dan Jacobson (https://www.jidanni.org/)
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along
# with this program; if not, write to the Free Software Foundation, Inc.,
# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
```

All the C code in beep is GPLv2 already, so beep already ships with a copy of the GPLv2.

The `Author`, `Created On`, `Last Modified On`, and `Update Count` lines could be removed then from my POV.

Reply:
Looks great! Say, doesn't your sed version need a copyright too?
Also be sure to move all this to a contrib/morse/ subdirectory. Thanks.

Reply:
All done in  [jidanni:patch-1](https://github.com/jidanni/beep/tree/patch-1).